documentation/config_file_puppetserver.md

```diff
@@ -32,36 +33,25 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
-    * `use-legacy-auth-conf`: Optional. The method to be used for authorizing access to the HTTP endpoints served by the "master" service. The applicable endpoints are listed in [Puppet v3 HTTP API](/puppet/latest/reference/http_api/http_api_index.html#puppet-v3-http-api).
-    * `environment-class-cache-enabled`:  Optional.  Used to control whether 
-    or not a cache is maintained in conjunction with the use of the 
-    [`environment_classes` API](./puppet-api/v3/environment_classes.html).  If
-    this setting is set to `true`, the cache will be maintained, enabling an 
-    Etag header to be returned for each GET request to the API and for 
-    subsequent GET requests using the prior Etag value in an If-None-Match 
-    header to return an HTTP 304 (Not Modified) with no response body when 
-    the class information available for an environment has not changed.  If 
-    this setting is set to `false` or is not specified, no cache will be 
-    maintained.  In this case, an Etag header is not returned for GET 
-    requests and the If-None-Match header for an incoming request is ignored,
-    meaning that every incoming request will involve parsing the latest 
-    available code for an environment from disk.  For more information, see 
-    the `environment_classes` API documentation. 
-    * `compile-mode`: Optional, experimental.  Used to control JRuby's "CompileMode", which may improve performance.  The default value is `off`, which is the most conservative value.  A value of `jit` will enable JRuby's "just in time" compilation of Ruby code.  A value of `force` will cause JRuby to attempt to pre-compile all Ruby code ahead of time.
+    * `use-legacy-auth-conf`: Optional. The method to be used for authorizing access to the HTTP endpoints served by the master service. The applicable endpoints are listed in [Puppet v3 HTTP API](/puppet/latest/reference/http_api/http_api_index.html#puppet-v3-http-api).
+    * `environment-class-cache-enabled`: Optional. Used to control whether the master service maintains a cache in conjunction with the use of the [`environment_classes` API](./puppet-api/v3/environment_classes.html). If this setting is set to `true`, Puppet Server maintains the cache, enabling an Etag header to be returned for each GET request to the API and for subsequent GET requests using the prior Etag value in an If-None-Match header to return an HTTP 304 (Not Modified) response with no body when the class information available for an environment has not changed. If this setting is set to `false` or is not specified, Puppet Server doesn't maintain a cache, an Etag header is not returned for GET requests, and the If-None-Match header for an incoming request is ignored. Puppet Server therefore parses the latest available code for an environment from disk on every incoming request. For more information, see the [`environment_classes` API documentation][].
```

Comment:
> If this setting is set to `true`, Puppet Server maintains the cache...

This sentence is really long and I'm finding it hard to follow, might be worth breaking it up into two or three sentences?